apps/omg_api/lib/state/transaction.ex

```diff
@@ -54,63 +54,95 @@ defmodule OMG.API.State.Transaction do
   @type currency :: Crypto.address_t()
 
   @doc """
-  Creates transaction from utxos where first output belongs to receiver and second belongs to owner of utxos
-  and the amount decreased by receiver's amount and the fee.
+  Creates transaction from utxo positions and outputs. Provides simple, stateless validation on arguments.
 
-  assumptions:
-   ```length(utxos) = 1 | 2```
+  #### Assumptions:
+   * length of inputs between 1 and @max_inputs
+   * length of outputs between 0 and @max_inputs
```

Comment:
1) I'm aware that this will not replace constant name with the actual value. I'll make it clear this is constant in this module... (suggestions?)
2) Burning the money.